Doc/library/asyncio-eventloop.rst

```diff
@@ -970,6 +970,9 @@ Watching file descriptors
    invoke *callback* with the specified arguments once *fd* is available for
    reading.
 
+   Any preexisting callback registered for *fd* is cancelled and replaced by
```

Comment:
hmm this is a bit subtle - the Handle is cancelled, but not the callback, eg if the callback is currently executing it can't be interrupted.

This should be obvious to anyone reading add_reader - but I'll have a think

Reply:
I think it's fine.